.circleci/config.yml

```diff
@@ -84,10 +100,21 @@ aliases:
       - checkout
       - *merge_pullrequest
       - *submodules
-      - *setup
       - *cache_restore
       - *stack_build
       - *cache_save
+  - &build_with_nix
+    docker:
+      - image: nixos/nix
+    steps:
+      # I can't get caching to work on Nix due to missing certificates. Doesn't
+      # seem to bad though, thanks to nix's binary cache of 99% of Clash's
```

Comment:
```suggestion
      # seem too bad though, thanks to nix's binary cache of 99% of Clash's
```

Reply:
I use [cachix for diagrams](https://github.com/cchalmers/diagrams-dev/blob/master/.travis.yml) if you're interested. I kinda nice because users don't have to build anything.

Reply:
Huh, interesting. I'll look into it (probably not for this PR though).